Docs: state data source queried upon time range change (#75639)
grafana/grafana: docs/sources/dashboards/variables/add-template-variables/index.md

1. In the **Data source** list, select the target data source for the query. For more information about data sources, refer to [Add a data source][].
1. In the **Refresh** list, select when the variable should update options.
   - **On Dashboard Load:** Queries the data source every time the dashboard loads. This slows down dashboard loading, because the variable query needs to be completed before dashboard can be initialized.

>    - **On Time Range Change:** Queries the data source every time the dashboard loads _and_ when the dashboard time range changes. Use this option if your variable options query contains a time range filter or is dependent on the dashboard time range.

Reviewer: ```suggestion
   - **On Time Range Change:** Queries the data source every time the dashboard loads and when the dashboard time range changes. Use this option if your variable options query contains a time range filter or is dependent on the dashboard time range.
```

>    - **On Time Range Change:** Queries the data source everytime the dashboard loads and when the dashboard time range changes. No matter which dashboard variable query option you choose, the query always runs at dashboard load. Use this option if your variable options query contains a time range filter or is dependent on the dashboard time range. 

Reviewer: ```suggestion
   - **On Time Range Change:** Queries the data source every time the dashboard loads _and_ when the dashboard time range changes. Use this option if your variable options query contains a time range filter or is dependent on the dashboard time range. 
```
